Loggie/Classes/LoggieManager.swift

```diff
@@ -63,7 +65,10 @@ public class LoggieManager: NSObject {
     }
 
     func add(_ log: Log) {
-        logs.append(log)
+        // Avoid changing logs array from multiple threads (race condition)
+        logQueue.async { [weak self] in
+            self?.logs.append(log)
```

Comment:
If access to the logs is supposed be atomic, should we also wrap the getter in this queue?